### PR TITLE
Use bufferFrom(x) instead of Buffer(x) b/c deprecated in Node 7

### PR DESCRIPTION
**Pull request by paulkernfeld**
Also doing a bit of shopkeeping

### CODE REVIEW

**Comment by paulkernfeld** [comment]
Using buffer-from module in accordance w/ your tweeted policy

**Comment by paulkernfeld** [comment]
Agh, looks like the buffer-from module does not work with the very old version of npm that Travis uses to test node 0.8. Can we remove node 0.8 from the .travis.yml file?

Build failure is [here](https://travis-ci.org/paulkernfeld/html-tokenize/jobs/178982458).

File: tokenize.js [hunk 1]
```diff
@@ -13,7 +13,7 @@ module.exports = function (opts) {
     var bufs = buffers();
     var tokenize = through(
         function (buf) {
-            if (typeof buf === 'string') buf = Buffer(buf);
+            if (typeof buf === 'string') buf = Buffer.from(buf);
```
Comment (juliangruber) [inline tokenize.js]: why not use `bufferFrom` here as well?

Comment (paulkernfeld) [inline tokenize.js]: That's a mistake on my part; I added a commit fixing this.

File: .gitignore [hunk 2]
```diff
@@ -0,0 +1 @@
+node_modules
```
Comment (juliangruber) [inline .gitignore]: substack doesn't want this in his modules, always does manual `git add`

**Comment by juliangruber** [comment]
otherwise LGTM!

File: .gitignore (hunk 2 above)
Comment (paulkernfeld) [inline .gitignore]: Oops, I didn't know that. Thanks!

File: index.js [hunk 3]
```diff
@@ -1,3 +1,4 @@
+const bufferFrom = require('buffer-from')
```
Comment (juliangruber) [inline index.js]: should be `var`, to be consistent with the rest of the file. missed this one..

Comment (paulkernfeld) [inline index.js]: Yep, good call

**Comment by jeromew** [comment]
@paulkernfeld could you squash the PR into meaningful commits to avoid the noise in the commit history ?

The end of 0.8 support will probably need a major version bump to 2.0.0. Do I understand semver correctly ?

**Comment by paulkernfeld** [comment]
@jeromew okay, squashed! Is it normal GitHub etiquette to squash as I go? Or to squash at the end?

Yeah, this should be a major version bump. To be precise, it's possible that this still works with node 0.8, it just doesn't work with old versions of npm (1.2.30). To avoid this, we could probably update buffer-from to support older npm. What do you think?

**Comment by jeromew** [comment]
@paulkernfeld the PR was good for me too except it was kind of hard to read. I must admit I don't know if there is a github etiquette on this.

I think we can drop 0.8 support on this.

**Comment by jeromew** [comment]
just published 2.0.0 thanks @paulkernfeld and @juliangruber

**Pull request merged**